File: packages/firestore/ios/Plugin/Classes/Results/GetCountFromServerResult.swift
import Capacitor

@objc public class GetCountFromServerResult: NSObject, Result {
    let count: Int

    init(_ count: Int) {
        self.count = count
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["count"] = count
        return result as AnyObject
    }
}

File: packages/firestore/android/src/main/java/app/capgo/capacitor/firebase/firestore/GetCountFromServerResult.java
package app.capgo.capacitor.firebase.firestore.classes.results;

import app.capgo.capacitor.firebase.firestore.interfaces.Result;
import com.getcapacitor.JSObject;

public class GetCountFromServerResult implements Result {

    private final long count;

    public GetCountFromServerResult(long count) {
        this.count = count;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("count", count);
        return result;
    }
}
